### PR TITLE
refactor on api structure start

### PR DESCRIPTION
**Pull request by RabbITCybErSeC**

### CODE REVIEW

**Review by lucamrgs — APPROVED** [review]
Looks nice and uniform to me!

File: pkg/api/playbook/playbook_api.go [hunk 1]
```diff
@@ -3,22 +3,32 @@ package playbook
...
+
+// a playbookhandler implements the playbook api endpoints is dependent on a database.
+type playbookHandler struct {
 	playbookRepo playbookrepository.IPlaybookRepository
 }
 
-// NewPlaybookController makes a new instance of playbookControler
-func NewPlaybookController(controller database.IController) *playbookController {
-	return &playbookController{playbookRepo: controller.GetDatabaseInstance()}
+// newplaybookcontroller makes a new instance of playbookcontroler
```
Comment (MaartendeKruijf) [inline pkg/api/playbook/playbook_api.go:29]: Update description to match capitalisation of function. Maybe even be more descriptive.

File: pkg/api/playbook/playbook_api.go [hunk 2]
```diff
@@ -3,22 +3,32 @@ package playbook
...
-type playbookController struct {
+var log *logger.Log
+
+type Empty struct{}
+
+func init() {
+	log = logger.Logger(reflect.TypeOf(Empty{}).PkgPath(), logger.Info, "", logger.Json)
+}
+
+// a playbookhandler implements the playbook api endpoints is dependent on a database.
```
Comment (MaartendeKruijf) [inline pkg/api/playbook/playbook_api.go:24]: Capitalise correctly

File: pkg/api/playbook/playbook_api.go [hunk 3]
```diff
@@ -3,22 +3,32 @@ package playbook
...
 
-// NewPlaybookController makes a new instance of playbookControler
-func NewPlaybookController(controller database.IController) *playbookController {
-	return &playbookController{playbookRepo: controller.GetDatabaseInstance()}
+// newplaybookcontroller makes a new instance of playbookcontroler
+func NewPlaybookHandler(controller database.IController) *playbookHandler {
+	return &playbookHandler{playbookRepo: controller.GetDatabaseInstance()}
 }
 
 // getAllPlaybooks GET handler for obtaining all the playbooks in the database and return this to the gin context in json format
```
Comment (MaartendeKruijf) [inline pkg/api/playbook/playbook_api.go:34]: Fix

File: pkg/api/playbook/playbook_api.go
Comment (MaartendeKruijf) [inline pkg/api/playbook/playbook_api.go:1]: Fix capitalisation for all descriptions in file

File: pkg/api/playbook/playbook_api.go [hunk 4]
```diff
@@ -31,10 +41,10 @@ func NewPlaybookController(controller database.IController) *playbookController
 //	@success		200	{array}		cacao.Playbook
 //	@failure		400	{object}	api.Error
 //	@Router			/playbook/ [GET]
-func (plabookCtrl *playbookController) getAllPlaybooks(g *gin.Context) {
+func (playbookHndlr *playbookHandler) GetAllPlaybooks(g *gin.Context) {
```
Comment (MaartendeKruijf) [inline pkg/api/playbook/playbook_api.go:44]:
```suggestion
func (handler *playbookHandler) GetAllPlaybooks(g *gin.Context) {
```

File: pkg/api/reporter/reporter_api.go
Comment (MaartendeKruijf) [inline pkg/api/reporter/reporter_api.go:1]: Fix description capitalisation

**Review by MaartendeKruijf — CHANGES_REQUESTED** [review]
Nice job!

**Review by MaartendeKruijf — APPROVED** [review]

**Pull request merged**